tabs/src/components/app/views/RSCList.jsx

```diff
@@ -22,43 +23,63 @@ export function RSCList() {
                 rscType = "/teams/" + context?.groupId;
             } else if (context?.chatId) {
                 rscType = "/chats/" + context?.chatId;
+            } else if (context){
+                setAlert(true);
             }
 
             if (context) {
```

Comment:
we already check for context here, can the else if be removed and just put setAlert in the next check?
```suggestion
            if (context) {
            setAlert(true);
```